### PR TITLE
AVI-5: Create UI template for Dashboard (HTML/CSS only)

### LINKED ISSUE #5: Create UI template for Dashboard (HTML/CSS only)
### Description

Create the UI template for the Dashboard page with only HTML and CSS. Use your initiative to design a simple dashboard. 

### To Do

- [x] HTML/CSS files for page design should go in the `template` folder

### Acceptance Criteria

- A simplified dashboard page with an overview of the number of employees
- Dashboard should have a sidebar and main area

### Screenshot for inspiration

NB: Screenshot is only for inspiration purpose, your design do not have to be exactly the same.

Image

### PR DESCRIPTION
**Pull request by Doyin02**
### What does this PR do

This is the UI for the dashboard created with only html and css.

Closes #5

### How should it be tested

- Go to your local development, navigate to template 
- Inside the template you will find a file named `dashboard.html`
- NOTE: Inside the template, there is a folder named styles
- Right-click on the `dashboard.html` file and open with live-server

### Screenshots

Screenshot (21)

### CODE REVIEW

**Comment by harry-olabs** [comment]
@Doyin02 would've been nice if you shared a screenshot on the new look.

**Comment by Doyin02** [comment]
will do so now

**Comment by harry-olabs** [comment]
@Doyin02, it is looking good, just requires some improvements. 
I have attached screenshots below to give you an idea what we are trying to achieve.

| Design | Skeleton |
|--------|----------|
|   Screenshot 2023-06-14 at 02 12 32 |  Screenshot 2023-06-14 at 11 13 13 |

**Comment by Doyin02** [comment]
The before and after of the dashboard

| Before | After |
| -------- | -------- |
|  Screenshot (21) |  
Screenshot (34) |

File: template/dashboard.html [hunk 1]
```diff
@@ -4,10 +4,116 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/dashboard.css">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat" rel="stylesheet">
```
Comment (harry-olabs) [inline template/dashboard.html]: Consider adding fonts for bold (:600,700,800,900)

File: template/styles/dashboard.css [hunk 2]
```diff
@@ -0,0 +1,333 @@
+body{
+  font-family: 'Montserrat', sans-serif;
+  background-color: 	#f9fff9;
+}
+body, html{
+  background-color: rgb(248, 244, 244);
+}
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Reset some general style for all elements:

```css
* {
  margin: 0;
  padding: 0;
  box-sizing: border-box;
}
```

File: template/styles/dashboard.css [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+.container{
+  display: flex;
+}
+.navbar {
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Let there be an empty line between each style block. Do the same everywhere

File: template/styles/dashboard.css [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+ .owner-section__name {
+  margin: 0;
+  font-size: 18px;
+  font-weight: bold;
+}
+
+.owner-section__role {
+  margin: 0;
+  color: #888;
+}  */
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: This must be a typo, remove it.

```suggestion
} 
```

File: template/styles/dashboard.css [hunk 5]
```diff
@@ -0,0 +1,333 @@
...
+
+.sidebar__menu {
+  list-style-type: none;
+  padding: 0;
+  margin-top: 10px;
+  margin-left:20px;
+
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Avoid leaving empty space at the end of each block

File: template/styles/dashboard.css [hunk 6]
```diff
@@ -0,0 +1,333 @@
...
+      justify-content: center;
+      align-items: center;
+  }
+  .fas-history{
+      background-color:#d15267;
+      color: white;
+      padding: 15px;
+      border-radius: 10px;
+      box-shadow: 0 4px 8px rgba(0, 0, 0, 0.7);
+  }
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: This is not properly formatted. Correct the indentation

File: template/styles/dashboard.css [hunk 7]
```diff
@@ -0,0 +1,333 @@
...
+
+  .first-area{
+      display:flex;
+      flex-direction:column;
+      margin-right:200px;
+      padding:30px;
+      justify-content: center;
+      
+      
+  }
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: This is not properly formatted. Correct indentation everywhere

File: template/styles/dashboard.css [hunk 8]
```diff
@@ -0,0 +1,333 @@
...
+  flex-direction: column;
+}
+.item1{
+ /* background-color:  rgb(248, 244, 244); */
+ display: flex;
+ flex-direction: column;
+ width: 370px;
+ height:500px;
+ margin-right:10px;
+ /* box-shadow: 0 4px 8px rgba(0, 0, 0, 0.7); */
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Remove if not used

File: template/styles/dashboard.css [hunk 9]
```diff
@@ -0,0 +1,333 @@
...
+  padding:0 40px;
+  margin-top: 150px;
+
+}
+.card-items-number-column{
+  display: flex;
+  flex-direction: column;
+}
+.item1{
+ /* background-color:  rgb(248, 244, 244); */
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Remove if not used

File: template/styles/dashboard.css [hunk 10]
```diff
@@ -0,0 +1,333 @@
...
+ 
+}
+
+.item2{
+  background-color: greenyellow;
+}
+/* .item3{
+  background-color: green;
+
+} */
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Remove if not used

File: template/dashboard.html [hunk 11]
```diff
@@ -4,10 +4,116 @@
...
+            <img src="images/head.jpg" alt="User Image">
+        </div>
+      </nav>
+    </nav>
+    <div class="container">
+        <div class="sidebar">
+            <div class="sidebar__card">
+                <img src="images/head.jpg" alt="logo">
+                <div class="sidebar__container">
+                  <h4><b>John Doe</b></h4>
```
Comment (harry-olabs) [inline template/dashboard.html]: You don't need to wrap the text with `<b>` tag, make use of css to make the font bold.

File: template/dashboard.html [hunk 12]
```diff
@@ -4,10 +4,116 @@
...
+                                <h4>Female</h4>
+                            </div>
+                        </div> 
+                    </div>
+                </div>
+                <div class="card item3">
+                    <div class="item3-container">
+                        <img src="images/head.jpg" alt="Avatar">
+                        <div class="item3-content">
+                            <h3><b>Welcome John</b></h3>
```
Comment (harry-olabs) [inline template/dashboard.html]: The same applies here. Make use of css to make the font bold.

File: template/dashboard.html [hunk 13]
```diff
@@ -4,10 +4,116 @@
...
+                                <i class="fas fa-history"></i>
+                                <h4>Employee</h4>
+                            </div>
+                        </div>
+                        <div>
+                            <h3>700</h3>
+                        </div>
+                    </div>
+                    <div class="card-content">
+                        <h3><b>Employee Structure</b></h3>
```
Comment (harry-olabs) [inline template/dashboard.html]: The same applies here. Make use of css to make the font bold.

**Review by harry-olabs — CHANGES_REQUESTED** [review]

**Comment by Doyin02** [comment]
## This is the updated styles for the dashboard UI template 
Screenshot (41)

File: template/styles/dashboard.css [hunk 14]
```diff
@@ -0,0 +1,373 @@
+*{
+  padding:0;
+  margin:0;
```
Comment (harry-olabs) [inline template/styles/dashboard.css:3]: @Doyin02 why did you exclude `box-sizing: border-box`?

File: template/styles/dashboard.css [hunk 15]
```diff
@@ -0,0 +1,373 @@
+*{
```
Comment (harry-olabs) [inline template/styles/dashboard.css:1]: There should be space after the astericks.

File: template/styles/dashboard.css [hunk 16]
```diff
@@ -0,0 +1,373 @@
...
+  padding-right: 10px;
+}
+
+.navbar__left-logo{
+  display: flex;
+  align-items: center;
+  width:40px; 
+  height:40px;
+  margin-right: 10px;
+
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: I did mention in a previous comment, avoid empty space at the end of the closing block

File: template/styles/dashboard.css [hunk 17]
```diff
@@ -0,0 +1,373 @@
...
+  height: 35px;
+  padding:0px 10px;
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.navbar__right{
+  padding:10px 30px;
+
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Remove empty line

File: template/styles/dashboard.css [hunk 18]
```diff
@@ -0,0 +1,373 @@
...
+.navbar__left-logo{
+  display: flex;
+  align-items: center;
+  width:40px; 
+  height:40px;
+  margin-right: 10px;
+
+}
+.navbar__left-logo img{
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: Let there be an empty line after each block. Go over each style block and space out each block.

File: template/styles/dashboard.css (hunk 14 above)
Comment (Doyin02) [reply template/styles/dashboard.css:3]: I have already included it

File: template/styles/dashboard.css [hunk 19]
```diff
@@ -0,0 +1,385 @@
+*{
+  padding:0;
+  margin:0;
+  outline: 1px dashed red;
```
Comment (harry-olabs) [inline template/styles/dashboard.css]: This is no longer required. Remove it 🔥

**Review by harry-olabs — CHANGES_REQUESTED** [review]

**Comment by Doyin02** [comment]
already removed the style that wasn't needed.

**Review by harry-olabs — APPROVED** [review]

**Pull request merged**